src/cf-layout.less

```diff
@@ -497,11 +497,11 @@
     }
 
     &__main {
-        .respond-to-range(801px, 899px, {
+        .respond-to-range(@bp-med-min, @bp-med-max, {
```

Comment:
This might be one where we should leave in a raw pixel value. Have you tested the results of making this change? It's going from a range of 801–899 to a range of 901–1020. Which may be okay, depending on where we're adding in the sidebar now.